### PR TITLE
387

### PR DESCRIPTION
**Pull request by pkra**

### CODE REVIEW

**Comment by pkra** [comment]
I'm not sure why two commits are listed. I had accidentally opened this to master, then closed, then switched to develop and re-opened.

**Comment by dpvc** [comment]
> I'm not sure why two commits are listed.

It's probably because I messed up the travis change and merged into master, and then had to do the same to develop.  The merge to develop technically isn't in master; even though it is the same change, the merge is a new commit.

I will make the 2.0.2 release once MathJax v2.7.3 is out, updating the package.json to call in the new version as well.

**Comment by pkra** [comment]
> I will make the 2.0.2 release once MathJax v2.7.3 is out, updating the package.json to call in the new version as well.

Thanks. I have a fix for #134 in the works but that can wait until after 2.0.2 (since it would bump to 2.1.0, right?)

**Comment by dpvc** [comment]
Yes, I think that is right.

File: test/issue219.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ tape('Basic Check: pass jsdom object to output', function(t) {
   }, function(data) {
     var window = new JSDOM().window;
     t.ok(data.htmlNode instanceof window.HTMLElement, 'htmlNode is an HTMLElement');
-    t.ok(data.svgNode instanceof window.HTMLElement, 'svgNode is an HTMLElement');
-    t.ok(data.mmlNode instanceof window.HTMLElement, 'mmlNode is an HTMLElement');
+    t.ok(data.svgNode instanceof window.SVGElement, 'svgNode is an HTMLElement');
+    t.ok(data.mmlNode instanceof window.Element, 'mmlNode is an HTMLElement');
```
Comment (dpvc) [inline test/issue219.js]: Shouldn't these read `svgNode is an SVGElement` and `mmlNode is an Element` instead of `HTMLElement` for both?

Comment (pkra) [reply test/issue219.js]: Yes. But it's merged now?

Comment (dpvc) [reply test/issue219.js]: Yes, I was adding this to remind myself to fix it in a new PR.

Comment (pkra) [reply test/issue219.js]: Thanks for clarifying.

Comment (dpvc) [reply test/issue219.js]: I've taken care of this now.

**Pull request merged**